tests/vmi_cloudinit_test.go

```diff
@@ -405,20 +395,10 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 					// Store cloudinit data as k8s secret
 					By("Creating a secret with network data")
-					secret := kubev1.Secret{
-						ObjectMeta: metav1.ObjectMeta{
-							Name:      secretID,
-							Namespace: vmi.Namespace,
-							Labels: map[string]string{
-								util.SecretLabel: secretID,
-							},
-						},
-						Type: "Opaque",
-						Data: map[string][]byte{
-							// The client encrypts the secret for us
-							"networkdata": []byte(testNetworkData),
-						},
-					}
+					secret := newTestSecret(secretID, vmi.Namespace, map[string][]byte{
+						// The client encrypts the secret for us
```

Comment:
what's the point of this repeated comment?
shouldn't it sit at the function definition?
is it needed at all?

Reply:
Comments removed all over